.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: Release
+on:
+  release:
+    types: [published]
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+        architecture: x64
+    - run: pip install nox==2019.11.9
+    - run: pip install poetry==1.0.5
+    - run: nox
+    - run: poetry build
+    - run: poetry publish --username=__token__ --password=${{ secrets.PYPI_TOKEN }}
```

Comment:
Before building and uploading to PyPI, some kind of version bumping should happen. Within [1], some details are outlined how to make this happen, like:
> Before creating the next release of your package, you need to bump the version of your package. Use `poetry version` to update the version declared in `pyproject.toml`:
> ```
> poetry version <version>
> ```
> You can pass the new version explicitly, or a rule such as `major`, `minor`, or `patch`.

I would say `poetry version minor` by default here.

[1] https://cjolowicz.github.io/posts/hypermodern-python-06-ci-cd/

Reply:
Ah, `poetry version` will be invoked on the local repository by a human, right? So, nevermind - please forget about my comment.

Reply:
### Git repository tagging?
When following the workflow "Uploading your package to PyPI" on [1], tagging the repository will happen "manually" through »Draft a new release« on GitHub, right?

[1] https://cjolowicz.github.io/posts/hypermodern-python-06-ci-cd/

Reply:
While the documentation says
> Enter the Git tag. By convention, these tags have the form v<version>.

I personally don't like the "v" prefixes on Git tags.